sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1524,6 +1524,12 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val PREFER_INTEGRAL_DIVISION = buildConf("spark.sql.function.preferIntegralDivision")
```

Comment:
After a second thought, I think we should not add a new behavior that is not SQL standard. If we only need it in tests, shall we make it clear in the config name? and make it an internal config.

Reply:
It seems that SQL standard does not explain this case(integral `/` integral ), different databases have different implementations.
![image](https://user-images.githubusercontent.com/5399861/62373377-3ca43280-b56c-11e9-90f7-b2e70965af96.png)

Reply:
OK so I'd like to treat it as an internal config that is only used to in the ported pgsql test cases. @wangyum can you send a follow-up PR? thanks!

Reply:
OK